lib/trample/condition.rb

```diff
@@ -20,12 +19,32 @@ class Condition
     attribute :fields, Array
     attribute :user_query, Hash
     attribute :transform, Proc, default: ->(_,_) { ->(val) { val } }
+    attribute :search_klass
+    attribute :lookup, Hash
 
     def initialize(attrs)
       attrs.merge!(single: true) if attrs[:name] == :keywords
       super(attrs)
     end
 
+    def lookup_autocomplete
```

Comment:
in the future let's try to keep method lines down by adding private methods, but I think this is fine for now